src/Makefile.in

```diff
@@ -463,14 +464,14 @@ CGAMEOBJS = $(1210_OBJ) $(369MM_OBJ) $(3SPOT_OBJ) $(ABALONE_OBJ) $(ACHI_OBJ) $(A
 			$(QLAND_OBJ) $(RCHECKERS_OBJ) $(RINFIN_OBJ) $(RUBIKSMAGIC_OBJ) $(SANTORINI_OBJ) \
 			$(SEEGA_OBJ) $(SHOGI_OBJ) $(SLIDEN_OBJ) $(SNAKE_OBJ) $(TILECHESS_OBJ) \
 			$(TOOTNOTTO_OBJ) $(TT2_OBJ) $(TTC_OBJ) $(TTTPPM_OBJ) $(TTTIER_OBJ) $(WIN4_OBJ) \
-			$(WINKERS_OBJ) $(WUZHI_OBJ) $(XIGUA_OBJ)
+			$(WINKERS_OBJ) $(WUZHI_OBJ) $(XIGUA_OBJ) $(TTT4_OBJ)
```

Comment:
kk

Reply:
I mean techincally, no. 

It looks like it's sorted at first, but then it kindof starts again with 1 appearing in the middle.
I'll just put next to the TTs, since you insist on it